CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.4.1 (Sep 22, 2021)
+
+* Add better structured logging to the LHM components (https://github.com/Shopify/lhm/pull/108)
```

Comment:
Is it structured logging tho? I thought we just log some text. Structured logging is more like logging an object, no?

Reply:
True. I have changed the description.